### PR TITLE
Power Down

### PR DESCRIPTION
**Pull request by mistikk**

### CODE REVIEW

File: src/components/checkbox/view/checkboxStyles.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import EStyleSheet from 'react-native-extended-stylesheet';
+
+export const getCheckBackground = isCheck =>
+  EStyleSheet.create({
+    backgroundColor: isCheck ? '$primaryBlue' : '#FFF',
```
Comment (ue) [inline src/components/checkbox/view/checkboxStyles.js]: awesome

File: src/screens/transfer/screen/powerDownScreen.js [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+
+  _renderIncomingFunds = (poweringDownFund, poweringDownVests, nextPowerDown) => (
+    <Fragment>
+      <Text style={{ color: 'green', fontSize: 20, marginVertical: 5 }}>
+        {`+ ${poweringDownFund} STEEM`}
+      </Text>
+      <Text style={{ color: 'red', fontSize: 15, marginVertical: 5 }}>
+        {`- ${poweringDownVests} VESTS`}
+      </Text>
+      <Text style={{ marginVertical: 5 }}>{nextPowerDown}</Text>
```
Comment (ue) [inline src/screens/transfer/screen/powerDownScreen.js]: these can move style file

File: src/screens/transfer/screen/powerDownScreen.js [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+  );
+
+  _renderDestinationAccountItems = () => {
+    const { destinationAccounts } = this.state;
+
+    if (destinationAccounts.length <= 0) {
+      return this._renderButton();
+    }
+    return (
+      <View>
```
Comment (ue) [inline src/screens/transfer/screen/powerDownScreen.js]: can be fragment

File: src/screens/transfer/screen/powerDownScreen.js [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+
+    if (!destinationAccounts.some(item => item.username === username)) {
+      destinationAccounts.push({ username, percent, autoPowerUp });
+      setWithdrawVestingRoute(currentAccountName, username, percent, autoPowerUp);
+      this.setState({
+        isOpenWithdrawAccount: false,
+        destinationAccounts,
+      });
+    } else {
+      Alert.alert('Opps!', 'same username');
```
Comment (ue) [inline src/screens/transfer/screen/powerDownScreen.js]: can move to translation file

File: src/screens/transfer/screen/powerDownScreen.js [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+      currentAccountName,
+      steemPerMVests,
+    } = this.props;
+    const { amount, steemConnectTransfer, isTransfering, isOpenWithdrawAccount } = this.state;
+    let path;
+    let poweringDownVests = 0;
+    let availableVestingShares = 0;
+
+    const poweringDown = !isEmptyDate(selectedAccount.next_vesting_withdrawal);
+    const nextPowerDown = parseDate(selectedAccount.next_vesting_withdrawal);
```
Comment (ue) [inline src/screens/transfer/screen/powerDownScreen.js]: can use get from lodash

File: src/utils/isEmptyDate.js [hunk 6]
```diff
@@ -0,0 +1 @@
+export default s => parseInt(s.split('-')[0], 10) < 1980;
```
Comment (ue) [inline src/utils/isEmptyDate.js]: can move `utils/time.js`

**Comment by mistikk** [comment]
@ue updated

**Comment by ue** [comment]
bitmoji

**Pull request merged**